code/modules/reagents/chemistry/recipes.dm

```diff
@@ -60,6 +60,10 @@
 	else
 		new /obj/effect/temp_visual/shockwave(T)
 		playsound(T, 'sound/effects/bang.ogg', 25, 1)
+	// PARADISE EDIT: Allow only a certain amount of atoms to be pulled per unit
+	var/units_per_atom = 5
```

Comment:
A `#define` would perhaps be more appropriate?

Reply:
Maybe, but its only scoped to one place and probably for the best to be like this to keep it in one place